tests/test_browser.py

```diff
@@ -2081,7 +2081,15 @@ def test_sdl2_key(self):
           event.initKeyEvent("keydown", true, true, window,
                              0, 0, 0, 0,
                              c, c);
-          document.dispatchEvent(event);
+          var prevented = !document.dispatchEvent(event);
+
+          //send keypress if not prevented
```

Comment:
is this a test for browser differences? if so let's document that. of not then I'm not sure of the motivation for the check, it would always be one way or the other?